editor/animation_track_editor.cpp

```diff
@@ -1663,27 +1663,27 @@ void AnimationTrackEdit::_path_entered(const String &p_text) {
 String AnimationTrackEdit::get_tooltip(const Point2 &p_pos) const {
 
 	if (check_rect.has_point(p_pos)) {
-		return TTR("Toggle this track on/off");
+		return TTR("Toggle this track on/off.");
 	}
 
 	if (path_rect.has_point(p_pos)) {
 		return animation->track_get_path(track);
 	}
 
 	if (update_mode_rect.has_point(p_pos)) {
-		return TTR("Update Mode (How this property is set).");
+		return TTR("Update Mode (How this property is set)");
```

Comment:
Why remove the dot here? It's inconsistent with other changes where you add a dot.

Reply:
There's no dot because the tooltip itself is not written like a phrase, but like a name ("Update Mode" instead of "Update mode.").